bika/lims/browser/worksheet/views/add_duplicate.py

```diff
@@ -47,9 +47,8 @@ def __call__(self):
             ar_uid = self.request.get('ar_uid', '')
             src_slot = [slot['position'] for slot in self.context.getLayout() if
                         slot['container_uid'] == ar_uid and slot['type'] == 'a'][0]
-            position = self.request.get('position', '')
             self.request['context_uid'] = self.context.UID()
-            self.context.addDuplicateAnalyses(src_slot, position)
+            self.context.addDuplicateAnalyses(src_slot)
```

Comment:
@xispa reminder: revert this change, I was not aware that there is a selector in "Add Duplicate" to allow the user to choose the slot to use. That selector has the next available slot option selected by default instead of "new slot" option. If a WSTemplate is assigned to the WS, better to select the latter option as default.